Makefile

```diff
@@ -63,6 +63,13 @@ conftest-test: ## Run unit tests with conftest
 fmt: ## Apply default formatting to all rego files. Use before you commit
 	@opa fmt . --write
 
+amend-fmt: fmt ## Apply default formatting to all rego files then amend the current commit
+	@git diff $$(git ls-files '*.rego')
+	@echo "Amend commit '$$(git log -n1 --oneline)' with the above diff?"
```

Comment:
I wonder if this prompt would be better displayed after running `git add` so we can also display which files are being added.

Reply:
The git diff does show that info. Also I'd rather not leave things staged if user wants to abort. It's a little clunky if there's no diff, since the "above diff" is blank, so it might be nice to skip everything unless there's a diff, but maybe it's good enough as is.

Reply:
Good point!